pgSchema.sql

```diff
@@ -0,0 +1,21 @@
+\timing ON
+
+CREATE TABLE IF NOT EXISTS books (
+  id SERIAL PRIMARY KEY,
```

Comment:
I was thinking that you could make set this to `NUT NULL` but I just checked the docs and setting `SERIAL` automatically adds the `NOT NULL` constraint.